### ISSUE TITLE
Scope out a way to browse/discover filing codes

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/docassemble-EFSPIntegration#153]
For form authors, it's not always easy to know what filing codes you need to use. It would be nice to have tools to let authors browse by topic and see what various county courts use as codes for that kind of filing.

Very similar terms sometimes are different.

**Comment by nonprofittechy** [comment]
> With https://github.com/SuffolkLITLab/EfileProxyServer/tree/hateos, you kinda can browse the codes for a particular court, it would be simple to make a DA interview that makes it easier to go through all that. But we'd have to scope it well, sometimes there are 15 filing codes that look exactly the same, and you need an expert to distinguish them.

Yeah, I think the idea would be to quickly get info that an on-the-ground expert could go and validate. But @mnewsted may be able to better spec

**Comment by BryceStevenWilley** [comment]
This is currently present in https://github.com/SuffolkLITLab/docassemble-EFSPIntegration/tree/codes_interview. It's probably not super useful yet; I'll give it one more look to see if there's anything low hanging to add, and then merge. 

TBH, the most useful thing I have found myself doing often is doing direct SQL queries on tables, without having to filter by case type, filing type, etc. Idk about adding those APIs directly to the server, since they'd be really heavy, and return back a lot of extra info, but it might be useful. Idk.

**Comment by mnewsted** [comment]
Thanks for working on this @BryceStevenWilley and @nonprofittechy . While I agree that it would be helpful to be able to easily browse filing codes, we've found that we need clerk/SME input to validate. Sometimes we can take our best guess looking at the SQL results as Bryce has done, but we've found that not all jurisdictions use the same codes in the same ways. Additionally, the other layers (case type, filing type, etc.) that Bryce mentioned can leave us with numerous viable options. All that is to say, it's often easier to get the clerk/SME input up front, then search out the exact codes. That said, having a better UI than a query prompt would be a fine addition.
